converters/fromZigbee.js

```diff
@@ -698,6 +698,44 @@ const converters = {
             }
         },
     },
+    ZNCLDJ11LM_curtain_genAnalogOutput_change: {
+        cid: 'genAnalogOutput',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            let running = false;
+            if (msg.data.data['61440']) {
+                if ((msg.data.data['61440'].toString()).startsWith('117440')
+            ||(msg.data.data['61440'].toString()).startsWith('591')) {
+                    running = true;
+                } else if (msg.data.data['61440']===0) {
+                    running = false;
+                }
+
+                return {presentValue: precisionRound(msg.data.data['presentValue'], 2), running: running};
```

Comment:
Whats does this present value mean?

Reply:
presentValue describes the current opening of the curtain. I changed this to "position" to align with https://www.home-assistant.io/components/cover.mqtt/